nyaa/routes.py

```diff
@@ -6,7 +6,8 @@
 from nyaa import torrents
 from nyaa import backend
 from nyaa import api_handler
-from nyaa.search import search_elastic, search_db
+from nyaa import api_torznab
+from nyaa.search import search
```

Comment:
You're right. In my local copy I already eliminated some of that duplicate code, and uses search( exclusively.